vinyl/functions.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * To connect to the database
+ *
+ * @param array 
+ * @return array associative array = $db
+ */
+function connectToDatabase(){
+    $db = new PDO('mysql:host=db; dbname=Vinyl', 'root', 'password');
+    $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+    return $db; 
+}
+
+/**
+ * To fetch all from the database 
+ *
+ * @param array 
+ * @return array = $data 
+ */
+function fetchAllDatabase($db){
```

Comment:
And here you can type hint $db as PDO as well, and you can return type hint this as : array